### ISSUE TITLE
Ionic 1 and XCode 10 iOS app redirect to background automatically

### ISSUE THREAD

**Issue by girish-fuluskar** [apache/cordova-ios#555]

**Comment by girish-fuluskar** [comment]
Hi,

I am facing this issue.

I am working on Ionic 1 app.

I am able to build .apk for latest version android phones and app is working fine.
With following command:
ionic cordova run ios – --buildFlag="-UseModernBuildSystem=0"
I am able to build iOS project from ionic and able to generate ipa file from xcode 10. But after installing .ipa on actual device (with iOS 12 and above), app open for a second and went into background.
Same .ipa file is working fine for older iOS devices with iOS 9 and 10.

Following are my setup details:

Ionic:

ionic (Ionic CLI) : 4.9.0 (/usr/local/lib/node_modules/ionic)
Ionic Framework : ionic1 1.3.3
@ionic/v1-toolkit : 1.0.17

Cordova:

cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
Cordova Platforms : android 7.1.4, ios 5.0.0
Cordova Plugins : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 2.3.2, (and 19 other plugins)

System:

ios-deploy : 1.9.2
ios-sim : 7.0.0
NodeJS : v10.15.0 (/usr/local/bin/node)
npm : 6.8.0
OS : macOS Mojave
Xcode : Xcode 10.1 Build version 10B61

Kindly advise.

**Comment by girish-fuluskar** [comment]
Hi,
I found the resolution for this issue.
It seems the issue was with CT signature issue.

Posting the resolution, so it might help someone.
In Keychain Access, I set the trust settings to "Use System Defaults" for the following certificates:

Apple Worldwide Developer Relations Certification Authority
Developer ID Certification Authority
iPhone Developer: {email}
iOS Distribution

After this, in XCode do clean build for the project. Also restart XCode.
